cmd/net-rpc-client.go

```diff
@@ -137,7 +137,7 @@ func (rpcClient *RPCClient) Call(serviceMethod string, args interface{}, reply i
 	// rpc.Client for a subsequent reconnect.
 	err := rpcLocalStack.Call(serviceMethod, args, reply)
 	if err != nil {
-		if err.Error() == rpc.ErrShutdown.Error() {
+		if err == rpc.ErrShutdown {
```

Comment:
Yes, because from golang spec: `Interface values are comparable. Two interface values are equal if they have identical dynamic types and equal dynamic values or if both have value nil. `